### DISCUSSION TITLE
Don't forget to set up XML Validation in Visual Studio Code

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#122] [Ideas] [answered]
During our June 26th Cartography & Curation workshop, @MattReimer mentioned the XML Validation. That is described [here](https://riverscapes.xyz/Tools/Technical_Reference/videos.html#xml-validation-in-visual-studio-code). All the @Riverscapes/riverscape-project-curators folks should have this set up for editing business logic.

**Comment by lauren-herbine** [comment, marked as answer]
It looks like it no longer requires the Java download, so if you haven't watched @philipbaileynar 's video yet, you can skip that part!
